src/components/Redoc/Redoc.tsx

```diff
@@ -22,10 +22,23 @@ import {
 import { SearchBox } from '../SearchBox/SearchBox';
 import { StoreProvider } from '../StoreBuilder';
 
+import { VersionSelector } from '../VersionSelector';
+
 export interface RedocProps {
   store: AppStore;
 }
 
+/** WILL REMOVE BEFORE MERGING!!! */
+/** Mock data that will be passed to store from cli */
+export const versionData = {
+  active: {
+    api_version: '2.0',
```

Comment:
Not a blocker, but if we need to use camel case for the versionData properties to avoid the mixed usage of camel case and snake case in the component, that should be fine.

(Definitely an oversight on my part when naming things in the spec, especially since this data should be passed from TS, not Python)

Reply:
Cool! Easy change for me. I was just going off the spec! Will change to snake case